block-storage-attacher/tests/e2e/e2e-tests/portworx_values.yaml

```diff
@@ -0,0 +1,12 @@
+kind: PersistentVolumeClaim
```

Comment:
Is this PVC used anywhere? I didnt find any place in e2e where its used.

Reply:
no action taken on this comment. Please check and revert.

Reply:
Removed the file